tests/structure/grid.test.ts

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai';
+
+import { default as $ } from 'jquery';
+(global as any).$ = $;
```

Comment:
We should fix this properly instead of hacking around it. Maybe we can remove this tests from this PR, and move it to a separate one where we also fix the issue?

Reply:
That would mean we can't write any tests for the structure viewer until we fix this, right?